docs/rest_api.md

```diff
@@ -522,46 +522,73 @@ The object will have at least the following fields:
                 - "label", the nic's label.
                 - "macaddr", the nic's mac address.
 		- "networks", a JSON object describing what networks are attached to the nic. The keys are channels and the values are the names of networks attached to those channels.
+        - "port", the port to which the nic is connected to or null if the nic
+          is not connected to any port.
+        - "switch", the switch that has the port to which the nic is connected
+          to or null if the nic is not connected to any port.
```

Comment:
Yeah, I think so -- otherwise someone might read this section and then wonder why the fields weren't showing up.